.github/workflows/publish_master.yml

```diff
@@ -70,6 +70,18 @@ jobs:
           context: .
           file: ./Dockerfile
           platforms: linux/amd64,linux/arm64
+          if: github.ref == 'refs/heads/master'
+            echo 'Building image from the ${{ github.ref }} branch'
+          run: |
+            docker build -t cova-master .
```

Comment:
Do you need to explicitly call `docker build`? Doesn't the `docker/build-push-action@v2` build it automatically?

One possible idea for how to do this in `master` that I had was build and push to a `covalent-test` repo in ECR (I think such a repo exists), then pull from `covalent-test` and run the `example_dispatch.py` using that image, then build and push to `covalent` repo in ECR

Reply:
Thanks. I will update this workflow with these steps

Reply:
@scottwn I can see the `covalent-test` repo in ECR and I wanted to use its variable name and not call the repo's url explicitly. Please do you know the variable name in secrets?

Reply:
Just added it as `AWS_ECR_TEST_REPO`

Reply:
@scottwn Thank you!